src/index.html

```diff
@@ -5,8 +5,437 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;300;400;500;700&display=swap"
+      rel="stylesheet"
+    >
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="page">
+    <header class="header">
+      <a href="#home">
+        <img
+         src="images/logo.png"
+         alt="moyo-logo"
+         class="header__logo-moyo"
+         >
```

Comment:
fix the code formatting. check all cases
```suggestion
        <img
          src="images/logo.png"
          alt="moyo-logo"
          class="header__logo-moyo"
         >
```